src/components/BodyContainer.js

```diff
@@ -0,0 +1,10 @@
+import CardContainer from "./CardContainer";
+
+const BodyContainer = ({members_data}) => {
+    console.log("laxmi" ,members_data)
+    return members_data.map(individual_data => {
+        return <CardContainer members_data = {individual_data} key = {individual_data.id}/>
```

Comment:
2nd return can be omitted from here..... for e.g:  map((individual_data) => (
                              <CardContainer members_data = {individual_data} key = {individual_data.id}/>))